e2etests/features/step_definitions/common.rb

```diff
@@ -1,7 +1,5 @@
-# frozen_string_literal:true
-
-Given(/^"([^"]*)" user$/) do |user|
```

Comment:
This step def will be needed for CMS

Reply:
When the tests need it, it can be added back, until then it makes the steps harder to read